script.js

```diff
@@ -9,30 +9,51 @@ if (window.XMLHttpRequest) {
   xmlhttp = new ActiveXObject("Microsoft.XMLHTTP");
 }
 
+function getStatus(){
+  if(window.location.hostname.includes("cyber.coffee")){
+    xmlhttp.open("GET", "status.json", true);
+    xmlhttp.send();
+  }
+  else if(window.location.hostname.includes("localhost")){
+    var rand = Math.floor((Math.random()*3) +1);
+    console.log(rand);
+    switch(rand){
+      case 1:
+        setStatus("fail");
+      break;
+      case 2:
+        setStatus("open");
+      break;
+      case 3:
+        setStatus("closed");
+      break;
+    }
+  } else {
+    xmlhttp.open("GET", "https://hodors.cyber.coffee/status.json", true);
```

Comment:
Mach das nicht so, die Seite könnte ja auch anders (z.B. im Metalab über die IPv4-Adresse oder im Internet über die IPv6-Adresse geladen sein). Was spricht gegen ```status.json```?

Reply:
Den JS "logic" Teil würde ich übrigens von diesem Pullrequest trennen und am besten dann gleich die Requests auf [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) umstellen.

Es stellt sich auch die Frage, ob man das nicht sowieso serverseitig renderen sollte. Das Python-Backend schreibt ja sowieso schon in das JSON-File hinein und könnte da auch gleich mit irgendeinem Templating das HTML machen, dann könnte man sich das ganze JS-Zeug sparen.